src/screens/LoginPage/LoginPage.tsx

```diff
@@ -233,8 +233,14 @@ function loginPage(): JSX.Element {
         <Row className={styles.row}>
           <Col sm={0} md={6} lg={7} className={styles.left_portion}>
             <div className={styles.inner}>
-              <PalisadoesLogo className={styles.palisadoes_logo} />
-              <p className="text-center">{t('fromPalisadoes')}</p>
+              <a
+                href="https://www.palisadoes.org/"
+                target="_blank"
```

Comment:
This file is unrelated to your PR, revert any change you made to this file.

Reply:
I reverted and modified as you said can you please review this again??